bokehjs/src/coffee/models/sources/ajax_data_source.coffee

```diff
@@ -26,32 +23,30 @@ export class AjaxDataSource extends RemoteDataSource
     @get_data(@mode)
     if @polling_interval
       @interval = setInterval(@get_data, @polling_interval,
-                              @mode, @max_size,
-                              @if_modified)
+                              @mode, @max_size, @if_modified)
 
   get_data : (mode, max_size=0, if_modified=false) =>
-    $.ajax(
-      dataType: 'json'
-      ifModified: if_modified
-      url : @data_url
-      xhrField :
-        withCredentials : true
-      method : @method
-      contentType : @content_type
-      headers : @http_headers
-    ).done((data) =>
-      if mode == 'replace'
-        @data = data
-      else if mode == 'append'
-        original_data = @data
-        for column in @columns()
-          data[column] = original_data[column].concat(data[column])[-max_size..]
-        @data = data
-      else
-        logger.error("unsupported mode: " + mode)
-      logger.trace(data)
-      return null
-    ).error(() ->
-      logger.error(arguments)
+    xhr = new XMLHttpRequest()
+    xhr.open(@method, @data_url, true)
+    xhr.withCredentials = false
```

Comment:
@bryevdv, I'm not sure about this. The original code uses `true`, but using `true` here fails with cross-domain errors. This isn't unexpected, because we don't send any credentials.